sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -744,22 +744,23 @@ void gnrc_rpl_send_DAO(gnrc_rpl_instance_t *inst, ipv6_addr_t *destination, uint
     void *ft_state = NULL;
     gnrc_ipv6_nib_ft_t fte;
     while(gnrc_ipv6_nib_ft_iter(NULL, dodag->iface, &ft_state, &fte)) {
+        gnrc_pktsnip_t *tmp;
         DEBUG("RPL: Send DAO - building transit option\n");
 
-        if ((pkt = _dao_transit_build(NULL, lifetime, false)) == NULL) {
+        if ((tmp = _dao_transit_build(pkt, lifetime, false)) == NULL) {
             DEBUG("RPL: Send DAO - no space left in packet buffer\n");
             return;
         }
-
+        pkt = tmp;
```

Comment:
Minor nitpick: This whole `tmp` to `pkt` sequence is not required anymore now that the release lines are gone right?

Reply:
Yes... will fix as well ^^"

Reply:
Doen